lib/SILOptimizer/Transforms/RedundantLoadElimination.cpp

```diff
@@ -1351,6 +1351,9 @@ SILValue RLEContext::computePredecessorLocationValue(SILBasicBlock *BB,
   }
 
   auto Val = Updater.getValueInMiddleOfBlock(BB);
+  if (!Val->use_empty()) {
```

Comment:
Is this needed given that `makeExistingValueAvailable` does the same as `makeNewValueAvailable` (see my other comment)?